zgrab_schema.py

```diff
@@ -205,7 +205,11 @@
         "value":Binary(),
         "length":Integer(),
         "lifetime_hint":Long()
-    })
+    }),
+    "crypto_variables":SubRecord({
```

Comment:
I'm alright putting it as a top level thing if we can find a descriptive name for it.